Application/mobile/app_code/lib/pages/container/details_state.dart

```diff
@@ -116,7 +115,7 @@ class ContainerDetailsState extends State<ContainerDetailsPage> {
               mainAxisAlignment: MainAxisAlignment.center,
               children: [
                 Text(
-                  '$_localization par $_owner',
+                  '$_city, $_address',
```

Comment:
Swap them so it's "Rue Graslin, Nantes"

Reply:
I think "Nantes, Rue Graslin" is better, isn't it ?